docs/src/api/class-browsercontext.md

```diff
@@ -363,6 +363,7 @@ await context.AddCookiesAsync(new[] { cookie1, cookie2 });
   - `httpOnly` ?<[boolean]> Optional.
   - `secure` ?<[boolean]> Optional.
   - `sameSite` ?<[SameSiteAttribute]<"Strict"|"Lax"|"None">> Optional.
+  - `topLevelSite` ?<[string]> For partitioned third-party cookies (aka [CHIPS](https://developer.mozilla.org/en-US/docs/Web/Privacy/Guides/Privacy_sandbox/Partitioned_cookies)), the top level site (scheme and host) used as the partition key. Optional.
```

Comment:
topLevelSite => partitionKey to stay closer to the documentation and algorithm, see https://github.com/privacycg/CHIPS